src/components/bot-challenge-protection/themes/news.json

```diff
@@ -0,0 +1,9 @@
+{
+	"reCaptcha-warning": {
```

Comment:
i dont think this will be applied cuz the classname above has two dashes `--warning`. I think that's our convention so this needs to be changed